### PR TITLE
Feature/7 map method

### PR DESCRIPTION
**Pull request by YaroslavAggressive**

### CODE REVIEW

File: backend/config_parser.py [hunk 1]
```diff
@@ -29,18 +31,19 @@ class Config:
...
 
-    population: int = field(default=0)
-    buildings: int = field(default=0)
-    map_length: int = field(default=0)
-    map_width: int = field(default=0)
-    minimal_wall_length: int = field(default=0)
-    iteration_constraint: int = field(default=0)
-    indent_from_borders: int = field(default=0)
-    wall_length_divider: int = field(default=0)
+    config_name: str = field(1, default=0)
```
Comment (yarnaid) [inline backend/config_parser.py]: is it ok default value 0 for str?

File: backend/config_parser.py [hunk 2]
```diff
@@ -29,18 +31,19 @@ class Config:
...
+    population: int = field(2, default=0)
+    buildings: int = field(3, default=0)
+    map_length: int = field(4, default=0)
+    map_width: int = field(5, default=0)
+    minimal_wall_length: int = field(6, default=0)
+    iteration_constraint: int = field(7, default=0)
+    indent_from_borders: int = field(8, default=0)
+    wall_length_divider: int = field(9, default=0)
```
Comment (yarnaid) [inline backend/config_parser.py]: do we need these default values? In protobuf you can omit them

File: backend/map_generation.py [hunk 3]
```diff
@@ -2,10 +2,18 @@
 from dataStructure.gRPC import Building, HumanState, BaseUnit, BuildingType
 from pure_protobuf.types import int32
 from itertools import count
+from pure_protobuf.dataclasses_ import message, field
+from dataclasses import dataclass
+from typing import List
 
 DUMMY_MAP_CONFIG_NAME = "tests/dummy_test_config.txt"
+SERIALIZATION_FILE = "C:/Users/tyryk/PycharmProjects/infection_spread/tests/test_serialization.txt"
```
Comment (yarnaid) [inline backend/map_generation.py]: Please, don't store your local paths in the code. Use relative paths and `pathlib`

File: backend/map_generation.py [hunk 4]
```diff
@@ -14,17 +22,30 @@ class ResearchMap:
     Parameters:
     ----------
 
+     config_name: str:
+        Filename for reading map settings
```
Comment (yarnaid) [inline backend/map_generation.py]: do we need this parameter in the message?

**Review by yarnaid — CHANGES_REQUESTED** [review]

File: backend/config_parser.py (hunk 1 above)
Comment (YaroslavAggressive) [reply backend/config_parser.py]: Done

File: backend/config_parser.py (hunk 2 above)
Comment (YaroslavAggressive) [reply backend/config_parser.py]: Done

File: backend/map_generation.py (hunk 4 above)
Comment (YaroslavAggressive) [reply backend/map_generation.py]: Done

File: backend/map_generation.py (hunk 3 above)
Comment (YaroslavAggressive) [reply backend/map_generation.py]: Done

File: backend/map_generation.py [hunk 5]
```diff
@@ -137,3 +155,7 @@ def create_dummy_map() -> ResearchMap:
                            Building(int32(4), 288, 418, BuildingType.HOUSE, int32(72), int32(76))]
     research_map.map_buildings = hardcoded_buildings
     return research_map
+
+
+ff = Config(DUMMY_MAP_CONFIG_NAME)
+print(ff)
```
Comment (yarnaid) [inline backend/map_generation.py]: please, remove this

File: tests/test_backend.py [hunk 6]
```diff
@@ -24,6 +24,7 @@ def test_intersection_check(test_data: Building) -> None:
     assert Building.intersection_check(first_building, second_building) == answer
 
 
+# testing dummy map
```
Comment (yarnaid) [inline tests/test_backend.py:27]: useless comment

**Review by yarnaid — CHANGES_REQUESTED** [review]

File: backend/map_generation.py (hunk 5 above)
Comment (YaroslavAggressive) [reply backend/map_generation.py]: Done

File: tests/test_backend.py (hunk 6 above)
Comment (YaroslavAggressive) [reply tests/test_backend.py:27]: Done

**Pull request merged**